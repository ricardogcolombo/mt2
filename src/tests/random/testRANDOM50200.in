50 200
1 49 39 38 87
1 2 73 36 64
1 48 113 39 30
1 19 31 31 75
1 2 77 34 52
1 29 13 44 24
1 41 92 47 102
1 6 78 7 119
1 44 10 1 11
1 5 67 41 3
1 22 23 30 103
1 27 122 4 57
1 22 39 5 80
1 3 98 11 80
1 22 1 2 86
1 37 72 8 129
1 10 24 39 111
1 23 130 21 16
1 39 91 20 56
1 28 90 12 95
1 15 28 10 127
1 13 32 14 41
1 17 29 37 87
1 41 115 29 70
1 41 24 7 26
1 45 3 24 63
1 22 83 45 73
1 5 59 32 12
1 50 40 8 23
1 44 128 39 76
1 4 74 13 104
1 23 106 34 76
1 20 92 6 51
1 39 61 37 46
1 4 82 45 52
1 1 129 8 124
1 49 23 34 111
1 42 10 40 118
1 16 25 39 115
1 17 53 48 8
1 27 45 17 59
1 15 12 9 95
1 17 111 8 75
1 49 62 1 130
1 12 57 16 127
1 10 3 14 4
1 50 48 30 12
1 2 21 34 107
1 12 12 14 19
1 41 75 27 119
1 32 10 9 29
1 5 56 41 111
1 13 48 47 101
1 45 54 35 103
1 16 93 27 10
1 24 114 46 106
1 46 30 8 27
1 35 0 14 40
1 2 97 28 23
1 29 45 41 1
1 40 106 33 44
1 44 47 13 1
1 3 64 45 21
1 7 90 9 25
1 39 56 27 14
1 33 20 49 44
1 23 100 21 23
1 29 27 47 116
1 23 0 15 38
1 21 39 47 10
1 6 129 28 122
1 1 72 39 95
1 4 74 36 47
1 20 98 16 21
1 26 93 50 18
1 46 12 23 89
1 46 59 3 7
1 21 7 13 98
1 38 6 42 89
1 47 7 26 35
1 16 21 46 43
1 9 117 34 103
1 40 52 29 0
1 12 40 9 82
1 11 35 13 118
1 14 18 34 13
1 36 97 30 127
1 2 86 9 77
1 20 98 24 112
1 25 34 29 48
1 26 129 10 43
1 4 62 30 126
1 5 55 26 3
1 18 20 37 49
1 11 49 42 64
1 43 49 13 6
1 30 49 23 54
1 15 122 3 14
1 45 60 18 70
1 26 100 37 107
1 7 91 35 83
1 46 64 35 67
1 8 96 31 59
1 5 26 15 71
1 23 46 27 10
1 35 10 41 75
1 38 126 3 57
1 41 30 24 63
1 17 105 36 91
1 1 10 26 11
1 23 71 6 126
1 23 94 21 28
1 9 36 21 1
1 31 14 17 24
1 37 47 3 117
1 31 85 25 43
1 12 34 47 5
1 23 124 41 15
1 35 54 39 72
1 28 23 43 106
1 11 13 31 34
1 49 70 26 112
1 12 71 9 127
1 21 34 49 101
1 7 48 49 12
1 13 1 48 31
1 27 77 28 32
1 35 104 18 106
1 34 3 16 113
1 49 19 25 23
1 35 46 18 82
1 35 63 26 57
1 3 67 40 43
1 29 100 19 24
1 16 3 23 104
1 24 19 48 92
1 48 99 39 27
1 46 47 38 61
1 41 118 2 76
1 49 56 20 19
1 50 31 18 108
1 33 113 25 5
1 19 69 38 10
1 12 36 45 25
1 39 99 42 67
1 26 116 12 73
1 5 14 47 125
1 42 82 38 93
1 27 4 24 28
1 5 58 50 118
1 11 79 43 2
1 15 124 38 42
1 28 54 6 48
1 17 98 48 48
1 48 19 15 24
1 21 91 6 50
1 24 56 50 61
1 9 18 28 83
1 8 36 33 126
1 3 61 2 95
1 30 52 39 116
1 11 17 21 63
1 38 9 50 54
1 27 93 3 46
1 41 44 38 63
1 30 106 42 86
1 16 16 38 50
1 6 76 50 60
1 2 97 39 118
1 7 0 18 127
1 3 68 40 10
1 48 38 21 87
1 18 97 25 20
1 35 42 13 10
1 30 98 1 89
1 2 127 50 115
1 11 2 38 56
1 18 122 4 97
1 24 46 18 93
1 34 2 22 41
1 42 20 30 115
1 13 127 17 96
1 26 22 14 94
1 26 105 23 97
1 16 13 50 79
1 29 61 47 67
1 12 30 28 72
1 7 53 10 77
1 12 91 34 105
1 25 40 15 9
1 16 100 3 66
1 47 45 36 89
1 23 83 10 81
1 22 88 20 124
1 39 69 27 67
1 37 39 39 113
1 36 7 44 69
1 47 50 39 53
1 37 61 16 125
1 40 109 22 37
