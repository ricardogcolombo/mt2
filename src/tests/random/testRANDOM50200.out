0.666667
0.333333
0.000000
0.500000
0.333333
0.666667
0.500000
0.545455
0.300000
0.500000
0.222222
0.500000
0.384615
0.500000
0.500000
0.545455
0.571429
0.250000
0.285714
0.375000
0.142857
0.700000
0.714286
0.600000
0.375000
0.600000
0.400000
0.800000
0.500000
0.363636
0.857143
0.500000
0.000000
0.666667
0.454545
0.400000
0.583333
0.571429
0.875000
1.000000
0.285714
0.333333
0.400000
0.800000
0.454545
0.555556
0.500000
0.500000
0.777778
0.500000
